netbox/templates/inc/filter_list.html

```diff
@@ -1,7 +1,7 @@
 {% load form_helpers %}
 {% load helpers %}
 
-<form action="." method="get">
+<form action="." method="get" onsubmit="return cleanGetUrl(this)">
```

Comment:
This is putting javascript in the templates, which as Jeremy said, we don't do. Use event listeners instead.

Docs: https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener